docs/impact/index.md

```diff
@@ -215,6 +197,47 @@ As of July 1, 2020, we have 183 people signed up for our
 [BIDS email list](https://forms.gle/JFo2aEkYbKY4EbmE6) and 412 members on our
 [google group](https://groups.google.com/forum/#!forum/bids-discussion).
 
+
+## How the BIDS team can help you
+
+If you are in the process of putting together a grant, **please email/message
+the pillar lead** that is most closely associated with your proposed grant so we
+may help support this. Our organization is structured into 3 pillars: standard,
+tools, and collaboration.
+
+The respective leads are:
+
+-   Stefan Appelhoff ([stefan.appelhoff@mailbox.org](mailto:stefan.appelhoff@mailbox.org)),
+-   Chris Markiewicz ([markiewicz@stanford.edu](mailto:markiewicz@stanford.edu))
```

Comment:
I think we should just switch the contact info to be the bids.maintenance@gmail.com address.

Reply:
Same for below on line 219